iso3166.min.d.ts

```diff
@@ -0,0 +1,20 @@
+export as namespace iso3166;
+
+interface Subdivision {
+  type: string;
+  name: string;
+}
+interface Subdivisions {
+  [code: string]: Subdivision;
+}
+interface Country {
+  name: string;
+  sub: Subdivisions;
+}
+
+export function country(name: string): Country;
+export function subdivision(type:string, name: string): Subdivision;
```

Comment:
según https://github.com/omegaup/iso-3166-2.js/blob/96d7d90157ac5e9f615ca0b1564f0902278a3778/functions.js#L21, esto se debería llamar

```suggestion
export function subdivision(country:string, code: string): Subdivision;
```

Reply:
listo